src/pages/activity/index.tsx

```diff
@@ -49,7 +50,20 @@ const Activity = ({ activities }: { activities: ActivityType[] }) => {
         )
         if (result.data && result.data?.length > 0) {
           pageView(`${router.asPath}?page=${updatedOffset}`)
-          const { data } = result
+          const data: ActivityType[] = []
+
+          result.data.map(item => {
+            data.push({
+              content: item.content,
+              datetime: item.datetime,
+              id: item.id,
+              ipfs: item.ipfs,
+              type: item.type,
+              wikiId: item.wikiId,
+            })
+            return null
+          })
```

Comment:
@OleanjiKingCode you can use  array returned by map instead of pushing to empty array

```suggestion
          const data: ActivityType[] = result.data.map(item => ({
              content: item.content,
              datetime: item.datetime,
              id: item.id,
              ipfs: item.ipfs,
              type: item.type,
              wikiId: item.wikiId,
            }))
```